lib/sshkit/scp.ex

```diff
@@ -36,11 +36,11 @@ defmodule SSHKit.SCP do
   ## Example
 
   ```
-  :ok = SSHKit.SCP.upload(conn, ".", "/home/code/sshkit", recursive: true)
+  :ok = SSHKit.SCP.upload(conn, "scp -t -r /home/code/sshkit", ".", recursive: true)
   ```
   """
-  def upload(connection, local, remote, options \\ []) do
-    Upload.transfer(connection, local, remote, options)
+  def upload(connection, command, source, options \\ []) do
```

Comment:
While at renaming: if `connection` is a `conn`, we could also go with that everywhere 🤔